services/rfq/relayer/chain/chain.go

```
@@ -74,6 +74,9 @@
 	gasAmount := big.NewInt(0)
 	var err error
 
+	//tmpdebug
```

golangci-lint (1.62.2)

commentFormatting: put a space between `//` and comment text

(gocritic)